External/TestDataSet/rawdata/sub-Sub1/perf/sub-Sub1_asl.json

```diff
@@ -0,0 +1,52 @@
+{
+	"Modality":"MR",
+	"MagneticFieldStrength":3,
+	"ImagingFrequency":127.744,
+	"Manufacturer":"Philips",
+	"ManufacturersModelName":"Ingenuity",
+	"SoftwareVersions":"3.2.2_3.2.2.1",
+	"MRAcquisitionType":"2D",
+	"PulseSequenceDetails":"v3.2.2 Product Sequence",
+	"ScanningSequence":"GR",
+	"SequenceVariant":"SK",
+	"ScanOptions":"FS",
+	"ImageType":["ORIGINAL","PRIMARY","M","FFE","M","FFE"],
+	"SeriesNumber":1901,
+	"AcquisitionTime":[14,18,22,26,30,34,38,42,46],
+	"AcquisitionNumber":19,
+	"AcquisitionVoxelSize":[3.4,3.4,4.5],
+	"SliceThickness":4.5,
+	"SpacingBetweenSlices":4.5,
+	"SAR":0.102452,
+	"EchoTime":0.010487,
+	"RepetitionTime":4.8,
+	"FlipAngle":90,
+	"CoilString":"SENSE-Head-8",
+	"PartialFourier":0.435484,
+	"PercentPhaseFOV":100,
+	"EchoTrainLength":27,
+	"PhaseEncodingSteps":62,
+	"PostLabelingDelay":2.025,
+	"AcquisitionMatrixPE":62,
+	"ReconMatrixPE":64,
+	"PixelBandwidth":3054,
+	"PhaseEncodingAxis":"i",
```

Comment:
Rename PhaseEncodingAxis to PhaseEncodingDirection = 'i'

Reply:
f430da8